examples/getting_started/pt/swarm_script_executor_cifar10.py

```diff
@@ -55,11 +55,11 @@
 
         # In swarm learning, each client acts also as an aggregator
         aggregator = InTimeAccumulateWeightedAggregator(expected_data_kind=DataKind.WEIGHTS)
-        job.to(aggregator, f"site-{i}")
+        job.to(aggregator, f"site-{i}", id="aggregator")
 
         # In swarm learning, each client uses a model persistor and shareable_generator
-        job.to(PTFileModelPersistor(model=Net()), f"site-{i}")
-        job.to(SimpleModelShareableGenerator(), f"site-{i}")
+        job.to(PTFileModelPersistor(model=Net()), f"site-{i}", id="persistor")
```

Comment:
I think it would be better to set the `aggregator_id` and `persistor_id` in `SwarmClientController` using `job.as_id()`. Then no id would need to be hardcoded.

Reply:
what does "as_id" mean? set_id?

Reply:
it returns the component's id

Reply:
changed to use job.as_id()